.changeset/@graphql-tools_merge-7588-dependencies.md

```
@@ -0,0 +1,5 @@
+---
+"@graphql-tools/merge": patch
+---
+dependencies updates:
+  - Updated dependency [`@graphql-tools/utils@^10.10.1` ↗︎](https://www.npmjs.com/package/@graphql-tools/utils/v/10.10.1) (from `^10.10.0`, in `dependencies`)
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 0; Actual: 2

(MD007, ul-indent)